BCnEncTests/AtcTests.cs

```diff
@@ -0,0 +1,59 @@
+using BCnEncoder.Decoder;
+using BCnEncoder.Encoder;
+using BCnEncoder.Shared;
+using BCnEncTests.Support;
+using Xunit;
+
+namespace BCnEncTests
+{
+	public class AtcTests
```

Comment:
Dds tests would be nice too. If you have the energy.

Reply:
So what I already have but instead of encoding to ktx, I encode to dds? Or do you mean loading an existing dds and assert its format?

Reply:
The first one.